language/move-analyzer/src/symbols.rs

```diff
@@ -1051,6 +1070,68 @@ impl Symbolicator {
         get_loc(&pos.file_hash(), pos.start(), files, file_id_mapping)
     }
 
+    /// Extracts the docstring (/// or /** ... */) for a given definition by traversing up from the line definition
+    fn extract_doc_string(&self, name_start: &Position, file_hash: &FileHash) -> String {
+        let mut doc_string = String::new();
```

Comment:
Nit. It would help readability a bit if you decreased the nesting level here. Please consider using the following pattern for the first two if-s:
```
let val = match ... {
  Some(v) => v,
  None => return doc_string,
}
```

Reply:
Refactored